Source/WebKit/WebProcess/WebPage/WebFoundTextRangeController.cpp

```diff
@@ -52,8 +52,21 @@ WebFoundTextRangeController::WebFoundTextRangeController(WebPage& webPage)
 {
 }
 
+#if PLATFORM(IOS_FAMILY)
```

Comment:
I've tried to avoid making this class have `IOS_FAMILY` specific logic. Can you try to remove `IOS_FAMILY` where possible? The shared logic could have an empty implementation.